docusaurus/docs/iOS/05-ui-cookbook/14-livestream-player.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Watching a livestream
+description: Documentation on how to play WebRTC livestream
+---
+
+As described in our livestream [tutorial](../../tutorials/livestream/), there are two ways of watching a livestream with StreamVideo's SDK: HLS and WebRTC. 
+
+Watching an HLS livestream can be done using Apple's native [AVPlayer](https://developer.apple.com/documentation/avfoundation/avplayer/).
+
+If you want to watch a WebRTC livestream, then you can either use our `LivestreamPlayer`, or build your own component.
+
+Our `LivestreamPlayer` provides a standard livestreaming experience:
+- shows a live indicator
+- shows the duration of the livestream
+- shows the number of participants
+- possibility to enter/leave full screen
+- possibility to pause/resume the livestream
+
+![Livestream Player](../assets/livestream-player.png)
+
+### Usage
+
+The `LivestreamPlayer` is a SwiftUI view that can be created with the livestream ID and the call type:
+
+```swift
+LivestreamPlayer(type: "livestream", id: "some_id")
+```
+
+You can show it, for example, via a `NavigationLink`, or as part of your own custom views.
+
+```swift
+NavigationLink {
+    LivestreamPlayer(type: "livestream", id: "vQyteZAnDYYk")
+} label: {
+    Text("Join stream")
+}
+```
+
+Make sure that the livestream id exists, and the call is not in backstage mode. Otherwise, the player will show a livestream not started error.
```

Comment:
Can we handle gracefully by:
- Displaying a message on screen
- Disable controls
- Hide the live label and the red dot

Reply:
done